### PR TITLE
merge

### PR DESCRIPTION
**Pull request by sazzeck**
Merge master branch and frontend branch

### CODE REVIEW

**Review by sazzeck — COMMENTED** [review]
all ok

**Review by semlish — APPROVED** [review]
cool

**Review by sazzeck — COMMENTED** [review]
style.css check

**Review by sazzeck — COMMENTED** [review]
index.html check

**Review by semlish — APPROVED** [review]

**Review by semlish — APPROVED** [review]

**Review by semlish — APPROVED** [review]

**Review by sazzeck — COMMENTED** [review]
all ok

File: templates/index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    
+</body>
+</html>
```
Comment (sazzeck) [inline templates/index.html:12]: where space?

**Comment by sazzeck** [comment]
all good

**Pull request merged**